config/defaults.go

```diff
@@ -0,0 +1,156 @@
+package config
+
+import (
+	"time"
+
+	"github.com/mrz1836/go-datastore"
+	"github.com/tonicpow/go-minercraft/v2"
+)
+
+// DefaultAppConfig is the default config for AppConfig
+var DefaultAppConfig = &AppConfig{
+	Authentication:     AuthConfigDefault,
+	Cache:              CacheDefault,
+	Db:                 DbDefaut,
+	Debug:              true,
+	DebugProfiling:     true,
+	DisableITC:         true,
+	GraphQL:            GraphqlDefault,
+	ImportBlockHeaders: "",
+	Monitor:            MonitorDefault,
+	NewRelic:           NewRelicDefault,
+	Nodes:              NodesDefault,
+	Notifications:      NotificationDefault,
+	Paymail:            PaymailDefault,
+	RequestLogging:     true,
+	Server:             ServerDefault,
+	TaskManager:        TaskManagerDefault,
+}
+
+// AuthConfigDefault is the default config for AuthenticationConfig
+var AuthConfigDefault = &AuthenticationConfig{
```

Comment:
Do we need it as a public var? do we use it somewhere else then just for initilialisation of DefaultAppConfig?

This ☝️ question also applies to all the variables below.

Reply:
Right, changed to private.